source/fundamentals/connection/tls.txt

```diff
@@ -0,0 +1,199 @@
+
+.. tabs::
+
+   .. tab:: ConnectionString
+      :tabid: connectionstring
+
+      To enable TLS/SSL on a connection with a :java-sync-api:`ConnectionString
+      <com/mongodb/ConnectionString.html>`, assign the connection string
+      parameter ``tls`` a value of ``true`` in the connection string passed to
+      ``MongoClients.create()``:
+
+      .. code-block:: java
+
+         MongoClient mongoClient = MongoClients.create("mongodb+srv://<user>:<password>@<cluster-url>?tls=true");
+
+   .. tab:: MongoClientSettings
+      :tabid: mongoclientsettings
+
+      To configure your ``MongoClient``'s TLS/SSL connection options using the
+      ``MongoClientSettings.Builder`` class, call the :java-sync-api:`applyToSslSettings()
+      <com/mongodb/MongoClientSettings.Builder.html#applyToSslSettings(com.mongodb.Block)>`
+      method. Set the ``enabled`` property to ``true`` in the ``SslSettings.Builder``
+      block to enable TLS/SSL:
+
+      .. code-block:: java
+
+         MongoClientSettings settings = MongoClientSettings.builder()
+                .applyToSslSettings(builder -> 
+                    builder.enabled(true))
+                .build();
+         MongoClient client = MongoClients.create(settings);
+
+.. _tls_configure-certificates:
+
+Configure Certificates
+----------------------
+
+Java applications that serve TLS/SSL requests require access to
+cryptographic certificates that prove identity for the application
+itself as well as other applications with which the application
+interacts. To configure access to these certificates in your application,
+you should use the JVM Trust Store and the JVM Key Store.
+
+.. _tls-configure-jvm-truststore:
+
+Configure the JVM Trust Store
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The JVM trust store saves certificates that securely identify other
+applications with which your Java application interacts. Using these
+certificates, your application can prove that the connection to another
+application is genuine and secure from tampering by third parties.
+
+An application that serves TLS/SSL requests needs to configure two
+JVM system properties to ensure that the client is able to validate
+the TLS/SSL certificate presented by a connected MongoDB instance.
+
+- ``javax.net.ssl.trustStore``: the path to a trust store containing the
+  certificate of the signing authority
+
+- ``javax.net.ssl.trustStorePassword``: the password to access the trust
+  store defined in ``javax.net.ssl.trustStore``
+
+You can create a trust store with the `keytool
+<http://docs.oracle.com/javase/8/docs/technotes/tools/unix/keytool.html>`__
+command line tool provided as part of the JDK:
+
+.. code-block:: console
+
+   keytool -importcert -trustcacerts -file <path to certificate authority file>
+            -keystore <path to trust store> -storepass <password>
+
+Configure the JVM Key Store
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The JVM key store saves certificates that securely identify your Java
```

Comment:
We should note that by default the server does not require the client to prove it's identity.  For example, to connect to Atlas this is not necessary.